ban/http/api.py

```diff
@@ -170,15 +172,22 @@ def get_resource(self, identifier):
 
         parameters:
             - $ref: '#/parameters/identifier'
+              name: identifier
```

Comment:
Not really, we have no description in schema.py for parameters identifier.
This is more precise...